rotors_gazebo/launch/fixed_wing_with_joy.launch

```diff
@@ -0,0 +1,36 @@
+<launch>
+  <arg name="uav_name" default="techpod"/>
+  <arg name="world_name" default="yosemite"/>
+  <arg name="enable_logging" default="false"/>
+  <arg name="enable_ground_truth" default="true"/>
+  <arg name="log_file" default="$(arg uav_name)"/>
+
+  <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find rotors_gazebo)/models"/>
+  <env name="GAZEBO_RESOURCE_PATH" value="${GAZEBO_RESOURCE_PATH}:$(find rotors_gazebo)/models"/>
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" value="$(find rotors_gazebo)/worlds/$(arg world_name).world"/>
+    <!-- <arg name="debug" value="true" /> -->
+    <arg name="paused" value="true" />
+    <!-- <arg name="gui" value="false" /> -->
```

Comment:
Yes, this was left there, as you thought, for one to know how to start gdb and how to turn the GUI off. Both of them I used frequently. But if no one else uses them, or they are obvious, I'm also open to remove them.